clang/unittests/Format/FormatTest.cpp

```diff
@@ -9256,7 +9347,7 @@ TEST_F(FormatTest, AlignsAfterOpenBracket) {
 
   Style.AlignAfterOpenBracket = FormatStyle::BAS_AlwaysBreak;
   Style.BinPackArguments = false;
-  Style.BinPackParameters = false;
+  Style.BinPackParameters = FormatStyle::BPPS_Never;
```

Comment:
```suggestion
  Style.BinPackParameters = FormatStyle::BPPS_OnePerLine;
```
and elsewhere below.